pkg/codegen/docs/templates/resource.tmpl

```diff
@@ -28,30 +33,38 @@
 {{ htmlSafe "{{% /choosable %}}" }}
 
 {{ htmlSafe "{{% choosable language nodejs %}}" }}
-{{ template "constructor_args" }}
+{{ template "constructor_args" .ConstructorParamsTyped.nodejs }}
 {{ htmlSafe "{{% /choosable %}}" }}
 
 {{ htmlSafe "{{% choosable language python %}}" }}
+{{/*
+    Constructing resources in python uses a different approach, which is why
+    we don't use the "constructor_args" and just hard-code two params here.
+*/}}
 <dl class="resources-properties">
     <dt class="property-required" title="Required">
-        <span>name</span>
+        <span>resource_name</span>
         <span class="property-indicator"></span>
+        <span class="property-type">str</span>
     </dt>
     <dd>The unique name of the resource.</dd>
     <dt class="property-optional" title="Optional">
         <span>opts</span>
         <span class="property-indicator"></span>
+        <span class="property-type">
+            <a href="#resource-arguments">object</a>
```

Comment:
This doesn't look right. `opts` in Python is typed as [`ResourceOptions`](https://www.pulumi.com/docs/reference/pkg/python/pulumi/#pulumi.ResourceOptions) and it should not be linking to the `resource-arguments` section, but rather to https://www.pulumi.com/docs/reference/pkg/python/pulumi/#pulumi.ResourceOptions